app/forms/courses/bulk_import_students_form.rb

```diff
@@ -33,7 +33,7 @@ def emails_must_be_valid
 
       return unless invalid
 
-      errors[:base] << 'One or more of the entries have an invalid email address'
+      errors[:base] << 'One or more entries have an invalid email address'
```

Comment:
Can you also move the strings to en.yml?

Reply:
@bodhish No, I'll leave that for a later change. This isn't really a _refactor_ of these files.